treeherder/etl/seta.py

```diff
@@ -30,11 +31,12 @@ def parse_testtype(build_system_type, job_type_name, platform_option, ref_data_n
     '''
     # XXX: Figure out how to ignore build, lint, etc. jobs
     # https://bugzilla.mozilla.org/show_bug.cgi?id=1318659
+    testtype = None
     if build_system_type == 'buildbot':
         # The testtype of builbot job can been found in 'ref_data_name'
         # like web-platform-tests-4 in "Ubuntu VM 12.04 x64 mozilla-inbound
         # opt test web-platform-tests-4"
-        return ref_data_name.split(' ')[-1]
+        testtype = ref_data_name.split(' ')[-1]
     else:
         # NOTE: Buildbot bridge tasks always have a Buildbot job associated to it. We will
         #       ignore any BBB task since we will be analyzing instead the Buildbot job associated
```

Comment:
we can remove the comment about Buildbot bridge (BBB)

Reply:
Ok. I didn't touch any of the buildbot related stuff. Do you want to remove it all?